app/utilities/schema_request_failed_exception.py

```diff
@@ -0,0 +1,2 @@
+class SchemaRequestFailed(Exception):
```

Comment:
I don't think we need a separate file for this.

Reply:
Removed separate file.